### ISSUE TITLE
[2017.11.13] Deep Hyperspherical Learning

### ISSUE THREAD

**Issue by unizard** [unizard/AwesomeArxiv#66]
#NIPS2017 spotlight

Institute: GIT, CMU, CAS, etc
URL: https://arxiv.org/pdf/1711.03189.pdf
Interest: 2
Keyword: Normalization Method ( BN, SN, BN+SN )
web: http://wyliu.com/
git: https://github.com/wy1iu/SphereNet (tensorflow) (to appear)

**Comment by unizard** [comment]
Deep Hypersher 라는 학술적 방법론으로 논문을 쭉쭉 쓰는 중... 

### Series works

- [SphereFace: Deep Hypersphere Embedding for Face Recognition](https://arxiv.org/abs/1704.08063), CVPR 2017, Git (pytorch, caffe2),

- Deep Hyperspherical Defense against Adversarial Perturbations, Technical Report, 2017.

**Comment by unizard** [comment]
### Interactive Machine Teaching

> 이 주제도 매우 맘에 듬. 칭화대 졸업하고 미국으로 넘어간 똑똑한 친구네.. 부럽

- [Iterative Machine Teaching](https://arxiv.org/abs/1705.10470), ICML 2017 (Oral), Git, Demo.
- [Towards Black-box Iterative Machine Teaching](https://arxiv.org/abs/1710.07742), Technical Report, 2017
